libathemecore/privs.c

```diff
@@ -265,6 +265,25 @@ soper_find_named(const char *name)
 
 	return NULL;
 }
+struct soper *
+soper_find_uid(const char *uid)
+{
+	struct soper *soper;
+	mowgli_node_t *n;
+	char lookup_uid[UIDLEN+1];
```

Comment:
I think this should be IDLEN+2

Reply:
how come

Reply:
oh null termination i forgot about that

Reply:
2 for ? and null terminator, IDLEN because UIDLEN is for s2s UIDs